executor/common_linux.h

```diff
@@ -128,11 +129,9 @@ struct nlmsg {
 	char* pos;
 	int nesting;
 	struct nlattr* nested[8];
-	char buf[1024];
+	char buf[4096];
 };
 
-static struct nlmsg nlmsg;
```

Comment:
Does this need to be moved in this commit?

Could it be a preparatory/cleanup commit?

Reply:
Created a separate commit.